docs/en/ingest-management/integrations/upgrade-integration.asciidoc

```diff
@@ -0,0 +1,147 @@
+[[upgrade-integration]]
+= Upgrade an {agent} integration
+
+IMPORTANT: An internet connection is required for {kib} to download integration
+packages from the Elastic Package Registry. Make sure the {kib} server can
+connect to `https://epr.elastic.co` on port `443`.
+
+//QUESTION: Do we want to mention air gapped environments here?
```

Comment:
I've decided no because I don't think we want to introduce the complexity of air gapped environments in this particular topic.